src/components/ui/Panel.jsx

```diff
@@ -111,6 +111,10 @@ export default class Panel extends React.Component {
   }
 
   holdResizer = (event, forceResize = false) => {
+    if (this.state.size === 'default') {
+      this._lastDefaultHeight = this.panelDOMElement.offsetHeight;
+    }
```

Comment:
This `offsetHeight` should not be saved if the event has been triggered during the transition to the `default` state (before its final height is reached).